migrations/sql/V2020.04.15_1703__ReportsConfigurePermission.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO "orderlyweb_permission" VALUES ('reports.configure');
```

Comment:
Turned out we hadn't actually added a settings permission for the allow_guest_user, just a route and controller (we require users.manage to set allow_guest_user), so I've added a reports configure permission for setting pinned reports instead.

Reply:
do you think `settings.manage` might be better? just think that it's not immediately clear what "configuring reports" means...